modules/administration-guide/pages/calculating-che-resource-requirements.adoc

```diff
@@ -90,10 +97,11 @@ Learn how to calculate resources, such as memory and CPU, required to run {prod}
 >s|5500 m
 >s|1060 m
 |===
+====
 
-. Sum up the resources required for each workspace, and multiply them by the expected running workspaces count.
+. Multiply the resources calculated per workspace by the number of workspaces that you expect all of your users to run simultaneously.
 
-. Sum up the requirements for the {prod-short} Operator, Operands, and {devworkspace} Controller
+. Calculate the sums of the requirements for the {prod-short} Operator, Operands, and {devworkspace} Controller.
```

Comment:
this part is ok, but I'm not sure what is bad with the current wording

Reply:
@ibuziuk, I'm afraid we can't use `sum up` when talking about arithmetic operations because `sum up` means something else: https://dictionary.cambridge.org/us/dictionary/english/sum-up

Reply:
ok, let's use the new version